src/libs/tmdk/src/commands/nuke.ts

```diff
@@ -36,9 +39,27 @@ export const builder: CommandBuilder<Options, Options> = (yargs) =>
     },
   });
 
+/**
+ * Helper function during workspace nuke for determining existance of workspace
+ * @param workspaceId TM workspace
+ * @returns promise boolean
+ */
+async function workspaceExists(workspaceId: string) {
+  try {
+    await aws().tm.getWorkspace({ workspaceId: workspaceId as string });
+    return true;
+  } catch (e) {
+    if (e instanceof ResourceNotFoundException) {
+      return false;
```

Comment:
We can generalize this method to just throw the ResourceNotFoundException. Unless there is case where we continue after seeing workspace do not exist, we can warn user and throw original error

Reply:
I think the method is better left as is because it allows for other unexpected errors to be caught and thrown, while serving its original purpose of "workspaceExists" returning a boolean instead of just throwing the error, unless I misunderstood your change